dev/merge_arrow_pr.py

```diff
@@ -680,6 +683,19 @@ def prompt_for_fix_version(cmd, issue, maintenance_branches=()):
         maintenance_branches=maintenance_branches
     )
 
+    if (issue.current_fix_versions and
+        issue.current_fix_versions != default_fix_version):
+        print("\n=== The assigned milestone is not the default ===")
+        print(f"Assigned milestone: {issue.current_fix_versions}")
+        print(f"Current milestone: {default_fix_version}")
+        print("Please ensure to assign the correct milestone.")
+        if issue.issue["milestone"].get("state") == 'closed':
+            print("The assigned milestone state is closed. Contact the ")
+            print("Release Manager if it has to be added to a closed Release")
+        print("Please ensure to assign the correct milestone.")
```

Comment:
Can we remove duplicated messages? (`Please ensure to assign the correct milestone.`)